packages/transaction-controller/src/types.ts

```diff
@@ -167,6 +167,11 @@ type TransactionMetaBase = {
    */
   firstRetryBlockNumber?: string;
 
+  /**
+   * Whether the transaction is the first time interaction.
+   */
+  firstTimeInteraction?: boolean;
```

Comment:
Do we also need a `firstTimeInteractionLoading` property so the UI can more easily differentiate loading and disabled?

Reply:
I think we don't need `firstTimeInteractionLoading ` because this alert level is warning so not a blocker alert. Please let me know if missing a point here